cmake/extensions.cmake

```diff
@@ -233,31 +269,25 @@ Please provide one of following: CONF_FILES")
 
     if(DEFINED FILE_SUFFIX)
       # Prepare search for pm_static_board@ver_suffix.yml
-      zephyr_build_string(filename
-                          BOARD ${ZEPHYR_FILE_BOARD}
-                          BOARD_REVISION ${ZEPHYR_FILE_BOARD_REVISION}
-      )
+      zephyr_build_string(filename ${additional_append})
```

Comment:
why the `additional_append` list building and passing here ?
A CMake function allows empty arguments, which in this case means that calling:
```
zephyr_build_string(filename 
    BOARD plank
    BOARD_QUALIFIERS
    BOARD_REVISION
)
```
is identical to call
```
zephyr_build_string(filename 
    BOARD plank
)
```
which again means that if `${ZEPHYR_FILE_BOARD_REVISION}` expands to nothing, then it's identical to not pass a value to the argument.

Reply:
Updated